src/main/java/org/broadinstitute/hellbender/tools/exome/GetBayesianHetCoverage.java

```diff
@@ -214,8 +284,8 @@ protected Object doWork() {
         /* determine the job type */
         HetCoverageJobType jobType = HetCoverageJobType.NONE;
```

Comment:
Which the enum and switch? why not simply call the appropriate ```run``` method within the appropriate if body statement?